CONTRIBUTING.md

```diff
@@ -106,16 +106,16 @@ Yes this is a lot, we intend to automate it later :)
 
 ### Publishing to npm
 
-Before publishing any package to `npm` perform the following steps
+Before publishing any package to `npm` perform the following steps:
 
+- Run yarn to ensure the lockfile is up to date
 - Format, lint, test, and build your package using the standard scripts
 - Ensure that your package has a sensible `.npmignore`
 - Ensure that the `main` and `types` keys in your package are set correctly
-- Bump the version number in `package.json` (consult a senior dev if unsure)
-- Move all `Unreleased` changes to a new header with the new version number
-- Run yarn to ensure the lockfile is up to date
+- Update package version (yarn <package-to-release> version <strategy>[patch | minor | major](https://semver.org/))
```

Comment:
Keep in mind that the version strategy command doesn't work with release candidate versions

Reply:
Good to know, updated